src/main/client/src/components/@layout/IndexLayout.tsx

```diff
@@ -12,6 +13,7 @@ export const IndexLayout: React.FC<IndexLayoutProps> = ({ background, children }
     <Flex direction="column" minHeight="100vh">
       <Navbar />
       <Box background={background} flex={1} pb={20}>
+        <Warning />
```

Comment:
Move it just after Navbar IMO

Reply:
Nope, that would break the flex layout

Reply:
Hm. Tried it but might overlooked something.

Reply:
Yeah you right, it won't create a tragedy if the Warning is after Navbar. It was just my thought, that it would break, if the Navbar was fixed and moving with scrolling

Reply:
I didn't want to put it after the navbar, because it would stretch out to the whole viewport. This seemed the least complicated solution.

Reply:
Okay